app/views/articles/show.html.haml

```diff
@@ -1,3 +1,5 @@
+.notice
+.alert
```

Comment:
I would add these classes to the notice and error divs you have in the application layout